scripts/migrate-to-bcd.ts

```diff
@@ -1,15 +1,48 @@
-import assert from "node:assert";
 import fs from "node:fs";
 import path from "node:path";
-import { fileURLToPath } from "node:url";
 
 import { fdir } from "fdir";
 import winston from "winston";
 import yargs from "yargs";
 
 import { features } from "../index.js";
 
+const argv = yargs(process.argv.slice(2))
+  .scriptName("migrate-to-bcd")
+  .usage("$0 [bcd-path]", "Migrate compat_features keys to BCD tags")
+  .env("MIGRATE_TO_BCD")
+  .option("bcd-path", {
+    describe: "Path to a mdn/browser-compat-data checkout",
+    type: "string",
+    nargs: 1,
+  })
+  .option("verbose", {
+    alias: "v",
+    describe: "Show more information",
+    type: "count",
+    default: 0,
+    defaultDescription: "warn",
+  })
+  .middleware((argv) => {
+    // Read `--bcd-path` option from BCD_PATH if set, for compatibility with BCD
+    // Collector's config style
+    if (process.env.BCD_PATH) {
```

Comment:
I'm assuming this line would override the `bcd-path` argument, so we'd need the following, right?

```suggestion
    if (process.env.BCD_PATH && !argv.bcdPath) {
```

Reply:
Yes, good catch!